packages/camera/tizen/src/camera_device.cc

```diff
@@ -38,8 +40,19 @@ static std::string CreateTempFileName(const std::string &prefix,
   return file_name;
 }
 
-static ExifTagOrientation ChooseExifTagOrientatoin(
-    OrientationType device_orientation, bool is_front_lens_facing) {
+bool IsValidMediaPacket(media_packet_h media_packet) {
+  tbm_surface_h surface = nullptr;
+  int ret = media_packet_get_tbm_surface(media_packet, &surface);
+  if (ret != MEDIA_PACKET_ERROR_NONE) {
+    return false;
+  }
+  return true;
+}
+
+}  // namespace
+
+ExifTagOrientation ChooseExifTagOrientatoin(OrientationType device_orientation,
```

Comment:
Is there a reason for not including this in the anonymous namespace after removing `static`?

Reply:
Oh... I thought I'd need it somewhere else someday, now that I think about it, I don't think there's any need for that.
I wrapped the functions in anonymous namespaces :)